### ISSUE TITLE
check-realms TestPackageInstall.testInstall can take very long to fail

### ISSUE THREAD

**Issue by martinpitt** [cockpit-project/cockpit#13519]
Witnessed here: https://209.132.184.41:8493/logs/pull-13509-20200207-145858-3d9f32b4-debian-testing/log.html

```
# testInstall (check_realms.TestPackageInstall)
#
[0;1;31mFailed to connect to bus: No such file or directory[0m
bash: line 1: [: : integer expression expected
Failed to kill unit packagekit.service: Unit packagekit.service not loaded.
Created symlink /etc/systemd/system/packagekit.service -> /dev/null.

DevTools listening on ws://127.0.0.1:10190/devtools/browser/14568e82-cb9d-4f4d-bf3a-50b77b73cb9c
[0207/152008.579817:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory
[0207/152008.589282:ERROR:viz_main_impl.cc(180)] Exiting GPU process due to errors during initialization
[0207/152008.608951:WARNING:gpu_process_host.cc(1213)] The GPU process has crashed 1 time(s)
CDP: {"source":"security","level":"error","text":"Refused to apply inline style because it violates the following Content Security Policy directive: \"default-src 'self' http://127.0.0.2:9091\". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.\n","timestamp":1581088811643.646,"url":"http://127.0.0.2:9091/cockpit/$b4cdc291fe3f601fa303631df6aa5d614e144275f5d0f626d5b3e304e74ec66b/shell/index.js","lineNumber":14,"stackTrace":"(minified)"}
> warning: Could not connect: No such file or directory
> warning: transport closed: disconnected
/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:233
        throw Error(`Frame ${frame} (${frameId}) has no executionContextId`);
        ^

Error: Frame cockpit1:localhost/system (92F47B2624B929E4776C703787E1046C) has no executionContextId
    at getFrameExecId (/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:233:15)
    at eval (eval at process.stdin.on.chunk (/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:333:29), <anonymous>:1:246)
    at Socket.process.stdin.on.chunk (/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:333:29)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:265:13)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at Pipe.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)

DevTools listening on ws://127.0.0.1:9496/devtools/browser/e978812a-6b79-43c3-998a-76ad0763cd1a
[0207/153511.524160:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory
[0207/153511.526570:ERROR:viz_main_impl.cc(180)] Exiting GPU process due to errors during initialization
[0207/153511.533938:WARNING:gpu_process_host.cc(1213)] The GPU process has crashed 1 time(s)
/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:230
        throw Error(`Frame ${frame} is unknown`);
        ^

Error: Frame cockpit1 is unknown
    at getFrameExecId (/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:230:15)
    at eval (eval at process.stdin.on.chunk (/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:333:29), <anonymous>:1:246)
    at Socket.process.stdin.on.chunk (/build/cockpit-project/bots/make-checkout-workdir/test/common/chromium-cdp-driver.js:333:29)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:288:12)
    at readableAddChunk (_stream_readable.js:265:13)
    at Socket.Readable.push (_stream_readable.js:224:10)
    at Pipe.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)

[... this repeats countless times, until ... ]

Traceback (most recent call last):
  File "test/verify/check-realms", line 793, in testInstall
    self.waitTooltip("realmd is not available on this system")
  File "test/verify/check-realms", line 780, in waitTooltip
    self.browser.wait(check)
  File "/build/cockpit-project/bots/make-checkout-workdir/test/common/testlib.py", line 364, in wait
    raise Error('timed out waiting for predicate to become true')
testlib.Error: timed out waiting for predicate to become true

not ok 191 testInstall (check_realms.TestPackageInstall) # duration: 1137s
```

**Comment by martinpitt** [comment]
The [same problem on testDialogTransition](https://209.132.184.41:8493/logs/pull-13507-20200207-140808-1115c550-debian-stable/log.html#183)

**Comment by martinpitt** [comment]
Fixed in #13526
